controllers/pullRequests.js

```diff
@@ -37,7 +37,8 @@ const getUserPRs = async (req, res) => {
  */
 const getStalePRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchStalePRs()
+    const { size, page } = req.query
```

Comment:
Can you add these query params in the schema for the routes?

Reply:
Sure, will do that.

Reply:
Done. Please check now

Reply:
![image](https://user-images.githubusercontent.com/56217868/116276049-c3b62700-a7a1-11eb-8625-0295df82bd75.png)